ppl/archive/chunk/metadata.go

```diff
@@ -0,0 +1,74 @@
+package chunk
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/brimsec/zq/pkg/bufwriter"
+	"github.com/brimsec/zq/pkg/iosrc"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zio/zngio"
+	"github.com/brimsec/zq/zng/resolver"
+	"github.com/segmentio/ksuid"
+)
+
+type Metadata struct {
+	First       nano.Ts
+	Last        nano.Ts
+	RecordCount uint64
+	Masks       []ksuid.KSUID
+	Size        int64
+}
+
+func (md Metadata) Chunk(dir iosrc.URI, id ksuid.KSUID) Chunk {
+	return Chunk{
```

Comment:
Because I love single-character receiver names and Metadata is on the large size for a value receiver.
```suggestion
func (m *Metadata) Chunk(dir iosrc.URI, id ksuid.KSUID) Chunk {
	return Chunk{
```

Reply:
Yes you do